### ISSUE TITLE
it's impossible to provide many `exclude` options

### ISSUE THREAD

**Issue by yegor256** [MikhailEpatko/hoc-badge-action#9]
Would be great to have this ability:

```
uses: actions/checkout@v4
with:
  exclude: 
    - vendor/** 
    - others/**/*.js 
```

Now it's impossible (however, it's possible in the `hoc` command line tool)

**Comment by MikhailEpatko** [comment]
> Would be great to have this ability:
> 
> ```
> uses: actions/checkout@v4
> with:
>   exclude: 
>     - vendor/** 
>     - others/**/*.js 
> ```
> 
> Now it's impossible (however, it's possible in the `hoc` command line tool)

According workflow [specification](https://github.com/SchemaStore/schemastore/blob/e7de43504da53808eb2a11de4f5baa64933e7176/src/schemas/json/github-workflow.json#L788-L802) arguments might be a string, not array.

Could you show me an example how to pass an array to  the `hoc` command line tool?
Unfortunately I didn't find such example in the hoc documentation.

**Comment by yegor256** [comment]
@MikhailEpatko you can just do this: `hoc '--exclude=vendor/**' '--exclude=others/**/*.js'`

Maybe this way:

```
uses: actions/checkout@v4
with:
  exclude: vendor/**,others/**/*.js 
```

**Comment by MikhailEpatko** [comment]
Issue solved in Release-3.0.6
